content/en/blog/_posts/2023-04-06-seccomp-profiles-edge.md

```diff
@@ -0,0 +1,281 @@
+---
+layout: blog
+title: "Having fun with seccomp profiles on the edge"
+date: 2023-04-06
+slug: seccomp-profiles-edge
+---
+
+**Author**: Sascha Grunert
+
+The [Security Profiles Operator (SPO)][spo] is a feature-rich operator for
```

Comment:
Could link to https://kubernetes.io/docs/concepts/extend-kubernetes/operator/ if you'd like to.

Reply:
Done